azconfig/client/pom.xml

```diff
@@ -81,7 +49,7 @@
         <dependency>
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-simple</artifactId>
-            <version>1.7.5</version>
+            <scope>test</scope>
         </dependency>
```

Comment:
As discussed offline - lets just bring in the API, not any specific impl.

Reply:
Fixed. Are there any other changes I should make?